docs/index.bs

```diff
@@ -3173,8 +3158,10 @@ spec: rfc7231; urlPrefix: https://tools.ietf.org/html/rfc7231
       <li>Invoke <a href="#run-service-worker-algorithm">Run Service Worker</a> algorithm with <var>installingWorker</var> as the argument.</li>
       <li><a>Queue a task</a> <var>task</var> to run the following substeps:
         <ol>
-          <li>Create a trusted event <var>e</var> that uses the {{InstallEvent}} interface, with the event type <code><a href="#service-worker-global-scope-install-event">install</a></code>, which does not bubble and is not cancelable.</li>
+          <li><a>Create an event</a> <var>e</var> with the {{InstallEvent}} interface.</li>
+          <li>Initialize <var>e</var>’s {{Event/type}} attribute to "<code>install</code>".</li>
           <li><a>Dispatch</a> <var>e</var> at <var>installingWorker</var>'s <a>environment settings object</a>'s <a for="environment settings object">global object</a> <var>globalObject</var>.</li>
```

Comment:
Ah.. It doesn't. It seems better to define a slot for the global object rather than the environment settings object. I don't have any other use of service worker's environment settings object apart from getting the global object. Also what'd be a good name for it? The service worker's ___ global object. Incumbent? But I think just "global object" would also be fine as a service worker won't have multiple global objects at the same time. Any suggestion?